upload/common/uploaded_file.py

```diff
@@ -148,6 +149,7 @@ def retrieve_latest_file_checksum_status_and_values(self):
             status = rows[0][0]
         return status, self.checksums
 
+    @retry(reraise=True, wait=wait_fixed(2), stop=stop_after_attempt(3))
```

Comment:
I think a lot of retries in this code base are missing reraise=True. This is definitely behavior we want from every retry. I've created https://github.com/HumanCellAtlas/upload-service/issues/285 and assigned it to you 🙂as a quick win. Good find on this param as as exceptions are difficult to read without this in place

Reply:
Sounds good!